src/components/Loader/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,87 @@
+import classNames from 'classnames';
+import { Person } from '../../../types';
+import { PersonLink } from '../../PersonLink';
+
+interface Props {
+  people: Person[],
+  currentElement: string,
+}
+
+export const PeopleList: React.FC<Props> = ({ people, currentElement }) => {
+  const findParent = (parent: 'mother' | 'father', person: Person) => {
+    const motherName = person.motherName ? person.motherName : '';
+    const fatherName = person.fatherName ? person.fatherName : '';
+
+    return parent === 'mother'
+      ? people.map(somePerson => somePerson.name)
+        .indexOf(motherName)
+      : people.map(somePerson => somePerson.name)
+        .indexOf(fatherName);
+  };
+
```

Comment:
Move outside from component to helper function.
Aslo instead of using map + indexOf - use Array.find()